src/enums/serviceMap.ts

```diff
@@ -123,6 +126,9 @@ export default {
   [services.apiGatewayHttpApi]: APIGatewayHttpApi,
   [services.apiGatewayResource]: APIGatewayResource,
   [services.apiGatewayRestApi]: APIGatewayRestApi,
+  [services.apiGatewayApiKey]: APIGatewayApiKey,
```

Comment:
Remember to add the new services to the `awsAccount` schema.graphql

Reply:
Done